en/docs/testing/unit-testing-in-build-pipeline.md

```diff
@@ -0,0 +1,28 @@
+# Unit Testing in Choreo Build Pipeline
+
+Choreo supports unit testing as part of the build pipeline, enabling users to validate their applications before deployment. This feature ensures that code changes are tested early, reducing the risk of defects in production.
+
+!!! note
+        Currently, Choreo only supports unit testing for the `WSO2 MI` buildpack.
+
+## How It Works
+
+### Develop tests
+
+In your source code, develop unit tests using standard testing libraries. For example, unit tests can be developed for a WSO2 MI project by following this [documentation](https://mi.docs.wso2.com/en/latest/develop/creating-unit-test-suite/).
```

Comment:
Suggestion:
You can create unit tests in your source code using standard testing libraries. For step-by-step instructions on creating unit tests for WSO2 Micro Integrator projects, see [Creating a Unit Test Suite].